templates/projects/clone_project.html

```diff
@@ -0,0 +1,11 @@
+{% extends "projects/base.html" %}
+{% from "_formhelpers.html" import render_form %}
+{% set active_page = "projects" %}
+{% set active_project  = "all" %}
+{% set active_link = "clone" %}
+{% block projectcontent %}
+<p>
+    Clone a job will copy all aspects of the current project except tasks. Please set the name and short name for the new job.
```

Comment:
In pybossa terminology, "project" can be used here instead of "job". Text can be rephrased to something following or similar.
> Cloning project will copy all possible settings except tasks, task runs and results from current project to the new/cloned project. Please set a different project name and project short name for the new project.